bigquery_etl/util.py

```diff
@@ -224,3 +292,60 @@ def coerce_result(*elements: Any) -> Generator[Any, None, None]:
             yield str(element)
         else:
             yield element
+
+
+def get_differences(exp, res, path="", sep=" / "):
```

Comment:
I would find this significanlty easier to read if it used the full names for the variables `expected` and `result`
```suggestion
def get_differences(expected, result, path="", sep=" / "):
```